storage/plugins/ncip-server/NcipServerPlugin.php

```
@@ -0,0 +1,1363 @@
+        $pageTitle = __('Log Transazioni NCIP');
+        require __DIR__ . '/../../../app/Views/layout.php';
+        $html = (string) ob_get_clean();
+        $response->getBody()->write($html);
+        return $response;
+    }
+
+    private function requireAdmin(): bool
+    {
+        return isset($_SESSION['user']) &&
+            in_array($_SESSION['user']['tipo_utente'] ?? '', ['admin', 'staff'], true);
+    }
```

PHPMD (2.15.0)

[error] requireAdmin accesses the super-global variable $_SESSION. (undefined)

(Superglobals)